_includes/releases/v22.2/v22.2.8.md

```diff
@@ -0,0 +1,88 @@
+
+<h3 id="v22-2-8-operational-changes">Operational changes</h3>
+
+- The `kv.trace.slow_request_stacks.threshold` [cluster setting](../v22.2/cluster-settings.html) can be used to attach available stack history from tracer snapshots to the trace of slow requests. [#99738][#99738]
+
+<h3 id="v22-2-8-db-console-changes">DB Console changes</h3>
+
+- New data is now automatically fetched every 5 minutes on the [Statement](../v22.2/ui-statements-page.html) and [Transaction](../v22.2/ui-transactions-page.html) pages. [#99271][#99271]
+- On DB console Stats pages, issuing a new request for stats while one is pending is now allowed and will replace the pending request. [#99271][#99271]
+
+<h3 id="v22-2-8-bug-fixes">Bug fixes</h3>
+
+- Prepared statements using placeholders in recursive [common table expressions](../v22.2/common-table-expressions.html) did not always re-optimize correctly after plugging in the parameters, which led to poor plan choices. This has been fixed. [#100327][#100327]
+- Fixed a bug where glob patterns that did not match tables in [`GRANT`](../v22.2/grant.html) or [`REVOKE`](../v22.2/revoke.html) statements would return an internal error with a confusing message instead of the `"no objects matched"` error. [#99437][#99437]
+- Since v22.2.0, CockroachDB could crash with the `"attempting to append refresh spans after the tracked timestamp has moved forward"` error in some rare cases (most likely when querying `pg_catalog` and `crdb_internal` virtual tables). This has been fixed. The workaround before upgrading would be to run `SET CLUSTER SETTING sql.distsql.use_streamer.enabled = false;`. [#99445][#99445]
+- When using `cluster_logical_timestamp()` as a [`DEFAULT`](../v22.2/default-value.html) expression, the node would sometimes crash. This has been fixed.[#99662][#99662]
+- The [`ALTER DEFAULT PRIVILEGES ... ON FUNCTIONS ...`](../v22.2/alter-default-privileges.html) statement is no longer allowed unless all nodes are running on v22.2 and the upgrade is finalized. This prevents a bug where executing this command in a mixed version cluster could cause nodes still running on v22.1 to crash. [#99845][#99845]
+- Fixed a bug where the [`TRUNCATE TABLE`](../v22.2/truncate.html) GC job can be stuck in running status if the table descriptor has been GCed. This happened because `TRUNCATE TABLE` creates new empty indexes, then replaces and drops the old indexes. The dropped indexes data are deleted and GCed within the `TRUNCATE TABLE` GC job, which are needed to see the table descriptor make progress. However, if the table data has been GCed, the job couldn't make progress. This patch makes the GC job able to handle the missing descriptor edge case and let the `TRUNCATE TABLE` GC job succeed. [#100137][#100137]
```

Comment:
This is hard to understand -- I tried an edit or two bug didn't make much progress. At the least it could use a copyedit, but it seems like maybe too much info for a RN.